### DISCUSSION TITLE
parse raw event

### DISCUSSION THREAD

**Discussion by ujaved** [ics-py/ics-py#399] [Q&A] [answered]
If you have a raw string containing just an event:
```
BEGIN:VEVENT
RECURRENCE-ID:20250721T143000Z
ATTENDEE;X-NUM-GUESTS=0;CN=foo@bar.com
DTSTAMP:20230822T201143Z
DTEND:20250721T150000Z
DTSTART:20250721T143000Z
SUMMARY:Busy
UID:86pk3a9799mmsr8jop33oioqma@google.com
END:VEVENT
```

What call do you use to parse this into an `Event`?

**Comment by N-Coder** [comment, marked as answer]
In v0.8 you can probably just do `Event.from_container(string_to_container(text))` using `from ics.contentline import string_to_container`

**Reply by ujaved** [reply]
yes but the latest release is 0.7.2

**Reply by N-Coder** [reply]
The alpha is on PyPi and the main branch also contains this. If you really want to find a way for 0.7 you'd probably need to figure something out from reading the code of that version.
